controllers/articlesController.js

```diff
@@ -156,4 +159,84 @@ export default class ArticlesController {
       return Response.send(res, STATUS.BAD_REQUEST, error, false);
     }
   }
+
+  /**
+ * @description users can bookmark articles for reading later.
+ * @static
+ * @param  {object} req - The request object
+ * @param  {object} res - The response object
+ * @return {object} - It returns the request response object
+ */
+  static async bookmark(req, res) {
```

Comment:
#### Nitpick
- Find a better name than `bookmark`, you could make it `createBookmark` or `bookmarkArticle` etc.

Reply:
done